pronouncing/__init__.py

```diff
@@ -52,6 +53,38 @@ def init_cmu(filehandle=None):
         for word, phones in pronunciations:
             lookup[word].append(phones)
 
+        rhymeDict = collections.defaultdict(list)
+        for word, phones in pronunciations:
+            rp = rhyming_part(phones)
+            if rp is not None:
+                rhymeDict[rhyming_part(phones)].append(word)
+
+def rhymesFast(word):
```

Comment:
this doesn't need to be a separate function in the library! if we're going to add a dictionary lookup for the rhymes, we might as well make the fast function the default, so you can go ahead and remove the old `rhymes()` function and rename this function to `rhymes()`.